lib/project_types/extension/commands/create.rb

```diff
@@ -3,6 +3,10 @@
 module Extension
   class Command
     class Create < ShopifyCli::SubCommand
+      DEVELOPMENT_SERVER_SUPPORTED_TYPES = [
+        "checkout_ui_extension",
```

Comment:
This is the only supported type, as it is the only template in the go executable currently. We can add to this as more templates are added.